client/landing/stepper/declarative-flow/internals/steps-repository/importer-wordpress/index.tsx

```diff
@@ -24,6 +33,23 @@ const ImporterWordpress: Step = function ( props ) {
 		migrateFrom
 	);
 
+	const customizedActionLabel = queryParams.get( 'customizedActionLabel' );
```

Comment:
We usually use the `ref` === query string to manage specific scenarios.

```suggestion
	const customizedActionLabel = queryParams.get( 'ref' ) === MIGRATION_FLOW;
```
I adding the ref on #93700

Reply:
Hm!

I tried to make it generic to not have references to other flows. But if we go with this approach, maybe I could also remove the label from the query string and use a specific label for the `MIGRATION_FLOW`. Do you think it aligns with the good practices in Calypso? The advantage is that the users wouldn't be able to change the link label just by changing the URL.

Reply:
> I tried to make it generic to avoid references to other flows.
Yes, ideally, we should not have references to other flows, but unfortunately, we don't have good mechanisms to solve this kind of scenario.   

My feeling is that having the CTA label configured via URL is more fragile, especially when we consider we need to support many languages with weird characters.

It also makes hard to evolve the button style, like adding strong o any other tag.

Reply:
Thank you for the suggestion, Gabriel! This change was applied in https://github.com/Automattic/wp-calypso/pull/93645/commits/dd94996950ff64606022bd824926c33fb8f78c28.

I also updated the `backToStep` here: https://github.com/Automattic/wp-calypso/pull/93645/commits/b92bc700b9750ab94d3dad5e24337b56143c1760